chromium_src/chrome/browser/process_singleton.h

```diff
@@ -28,7 +28,7 @@
 
 namespace base {
 class CommandLine;
-}
+}  // namespace base
```

Comment:
Majority of forward declarations in Chromium seem to not include the `// namespace xxx`, but I could not find any documented rules about this, and there are some places having mixed style:
https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/web_contents.h;l=51-80

But I'm good unifying the style.